__tests__/app.test.js

```diff
@@ -80,6 +80,26 @@ describe("/api/articles", () => {
           expect(article).toMatchObject(article7);
         });
     });
+    test("Status 200: responds with corresponding article object to requested id which includes a comment_count", () => {
+      return request(app)
+        .get("/api/articles/5")
+        .expect(200)
+        .then(({ body }) => {
+          const { article } = body;
+          const article5 = {
+            article_id: 5,
+            title: "UNCOVERED: catspiracy to bring down democracy",
+            topic: "cats",
+            author: "rogersop",
+            votes: 0,
+            article_img_url:
+              "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+            created_at: "2020-08-03T13:14:00.000Z",
+            comment_count: "2",
```

Comment:
For this one, we shouldn't be just adding comment_count to the test here, we shouldn't have to amend any earlier tests to make them pass. We should just have a new test to check that `comment_count` has been added 👍 
I think this test should still pass when you remove `comment_count`, as you've used `toMatchObject` which is a flexible matcher ⭐ 😄